pootle/templates/browser/overview.html

```diff
@@ -75,10 +75,18 @@ <h4 class="clear">{{ item.title }}</h4>
 {% endblock %}
 
 {% block pre_content %}
-{% if announcement %}
+{% if announcement or upload_form or display_download %}
```

Comment:
Yes, we can use overextends. `import_export/_sidebar.html` would then overextend a new `sidebar_content` block.